packages/angular_devkit/build_optimizer/src/transforms/prefix-functions.ts

```diff
@@ -78,9 +76,15 @@ export function findTopLevelFunctions(parentNode: ts.Node): Set<ts.Node> {
         topLevelFunctions.add(node);
       } else if (ts.isCallExpression(innerNode)) {
         let expression: ts.Expression = innerNode.expression;
+
+        if (isHelperName(expression.getText())) {
```

Comment:
getText can be expensive.  Better to check for an identifier and then access it’s text property.